mitodl/mitxonline
# add financial aid models to admin and load country income thresholds

#### Pre-Flight checklist


#### What are the relevant tickets?
fixes #559 

#### What's this PR do?
adds financial add models to admin and load country income thresholds

#### How should this be manually tested?
Just run the management command and that should load all the threshold incomes for each country
`docker-compose exec web python manage.py load_country_income_thresholds flexiblepricing/data/country_income_thresholds.csv`

#### Screenshots (if appropriate)
<img width="753" alt="Screenshot 2022-05-17 at 18 02 03" src="https://user-images.githubusercontent.com/4043989/168817121-67a7e763-6958-44bc-8e54-23ed585974fc.png">
<img width="757" alt="Screenshot 2022-05-17 at 18 02 09" src="https://user-images.githubusercontent.com/4043989/168817140-ecf547a5-65c0-4a73-b460-d99aab428c28.png">
